src/resources/Global/OrganizationMonitoring/OrganizationMonitoring.ts

```diff
@@ -0,0 +1,12 @@
+import Resource from '../../Resource.js';
+import {OrganizationMonitoringEndpointUsageModel} from './OrganizationMonitoringInterfaces.js';
+
+export default class OrganizationEndpointUsage extends Resource {
```

Comment:
Lets use the same name for the class and the file name.

```suggestion
export default class OrganizationMonitoring extends Resource {
```